doc/ping.xml

```diff
@@ -204,6 +204,19 @@ xml:id="man.ping">
           kernel.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>
+          <option>-e</option>
+        </term>
+        <listitem>
+          <para>Set identification field of the ECHO_REQUEST.
+          This implies SOCK_RAW option since setting identification
+          on SOCK_DGRAM doesn't work right now. The value of
+          the field may be printed with
+          <option>-v</option> option.
```

Comment:
This is the main concern I have, because on ICMP datagram socket this means requiring root. But there is no other way, it would not work on distros which switched to ICMP datagram socket.

Also, please use `<emphasis remap="I">raw socket</emphasis>` and `<emphasis remap="I">ICMP datagram sockets</emphasis>` - keep used terminology and formatting

Reply:
Nowadays you don't need root, you just need CAP_NET_RAW on the binary. I'm not sure if it is set on ping utility by package mainteiners though. And I don't feel I have enough knowledge about security to suggest it to maintainers if it's not set.
I read man on SOCK_DGRAM and I didn't find a way to set Identification field, and it gets overriden by kernel. So I used SOCK_RAW.

Reply:
formatting fixed